src/Http/Http.Extensions/src/HttpResponseJsonExtensions.cs

```diff
@@ -18,6 +18,8 @@ public static partial class HttpResponseJsonExtensions
 {
     private const string RequiresUnreferencedCodeMessage = "JSON serialization and deserialization might require types that cannot be statically analyzed. " +
```

Comment:
@brunolins16 this will all be undone with your changes right?

Reply:
I don't believe, the unsafe API will still need to be attributed, however, RDF + MVC should not call them anymore.